packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+
+// notes:
```

Comment:
@KristinLBradley @alex-ju I put comments in here for context, because there wasn't prior art to refer to for this. Once we're all on the same page, I'm fine to remove these if that's what we want.